apps/android/app/src/main/java/chat/simplex/app/model/SimpleXAPI.kt

```diff
@@ -100,6 +104,15 @@ class AppPreferences(val context: Context) {
       set = fun(value) = sharedPreferences.edit().putString(prefName, value).apply()
     )
 
+  private fun mkInstantPreference(prefName: String, default: Instant?): Preference<Instant?> =
```

Comment:
```suggestion
  private fun mkDatePreference(prefName: String, default: Instant?): Preference<Instant?> =
```

the name is confusing otherwise